roadmap/2020/MLOpsRoadmap2020.md

```diff
@@ -26,7 +26,7 @@ Terry Cox, Bootstrap Ltd <terry@bootstrap.je>
 
 MLOps could be narrowly defined as "the ability to apply DevOps principles to Machine Learning applications" however as we shall see shortly, this narrow definition misses the true value of MLOps to the customer. Instead, we define MLOps as “the extension of the DevOps methodology to include Machine Learning and Data Science assets as first class citizens within the DevOps ecology”.
 
-MLOps should be viewed as a practice for consistently managing the ML aspects of products in a way that is unified with all of the other technical and non-technical elements necessary to successfully commercialise those products with maximum potential for viability in the marketplace.
+MLOps should be viewed as a practice for consistently managing the ML aspects of products in a way that is unified with all of the other technical and non-technical elements necessary to successfully commercialise those products with maximum potential for viability in the marketplace. This includes DataOps, too, as machine learning without complete, consistent, semantically valid, correct, timely, and unbiased data is [pointless](https://databaseline.tech/lean-dml-operations/#why-dmlops) or leads to flawed solutions that can exacerbate built-in biases (cf. [Weapons of Math Destruction](https://weaponsofmathdestructionbook.com/)).
```

Comment:
Perhaps this could be reworded to be less emotive.

I would like group consensus on whether we will accept links in the roadmap document. My preference is that we do not do so as we cannot guarantee that the remote content will remain as reviewed. Selective quotes with sources listed as footnotes are preferred.

Reply:
I like it it mostly - first stentence is a bit long, and yeah perhaps a bit less emotive at the end, something like  "problematic" instead of "pointless" ?

Reply:
I did not touch the first sentence by the way. The rest is now rephrased as suggested.